native/windows/SwrveUnityBridge/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following 
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("SwrveUnityBridge")]
+[assembly: AssemblyDescription("Swrve is a single integrated platform delivering everything you need to drive mobile engagement and create valuable consumer relationships on mobile. This native Windows UWP SDK will enable your app to use all of these features.")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("Swrve Mobile Inc")]
+[assembly: AssemblyProduct("SwrveUnityBridge")]
+[assembly: AssemblyCopyright("Copyright Swrve Mobile Inc or its licensors © 2016")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+// Version information for an assembly consists of the following four values:
+//
+//      Major Version
+//      Minor Version 
+//      Build Number
+//      Revision
+//
+// You can specify all the values or you can default the Build and Revision Numbers 
+// by using the '*' as shown below:
+// [assembly: AssemblyVersion("1.0.*")]

```

Comment:
~~ (pendatic) but for the comment you could use the multiline comment /** */ instead